@xen-orchestra/backups/_runners/Metadata.mjs

```diff
@@ -100,17 +105,17 @@ export const Metadata = class MetadataBackupRunner extends Abstract {
                     schedule,
                     settings,
                   }).run()
-              )
+              ).catch(noop)
             )
           )
         }
 
         if (job.xoMetadata !== undefined && settings.retentionXoMetadata !== 0) {
           promises.push(
-            runTask(
+            Task.run(
```

Comment:
The only difference using `runInside` would make is that the task would stay pending if it doesn't have an error, so it's not a solution.
If we want the backup to fail when one subtask fails, we would have to remove the `.catch(noop)`, but it would need some testing as it may have other consequences.